src/atoms/InputField/InputField.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import "./InputField.css"
+
+const InputField = ({inputType, inputValue, onInputChange}) => {
```

Comment:
Refer this comment : https://github.com/Sg1211/todo-app-react/pull/1/files#r906078827